milatools/utils/remote_v1.py

```diff
@@ -19,7 +19,7 @@
 from fabric import Connection
 from typing_extensions import Self, TypedDict, deprecated
 
-from .utils import (
+from ..cli.utils import (
```

Comment:
Hmm?

Reply:
This used to be a relative import from `milatools.cli.utils` (`.utils` relative to where the file was placed, at `milatools.cli.remote.py`).
Now that the file is under `milatools.utils.remote_v1.py`, the relative import becomes `from ..cli.utils import (...)`